ra/ra.go

```diff
@@ -377,12 +382,25 @@ func NewValidationProfiles(defaultName string, configs map[string]*ValidationPro
 			}
 		}
 
+		var identifierTypes []identifier.IdentifierType
+		for _, identType := range config.IdentifierTypes {
+			identifierTypes = append(identifierTypes, identifier.IdentifierType(identType))
+		}
```

Comment:
No need to convert from string if you use identifier.IdentifierType in the config object as I suggest above.